src/components/Pagination/Pagination.tsx

```diff
@@ -1,33 +1,47 @@
 import Link from "next/link";
 
 import { ArrowLeftSmallIcon, ArrowRightSmallIcon } from "@/components/icons";
-import { FIESTA_ENDPOINTS } from "@/config";
 import { serialize } from "@/lib/searchParams";
 import { cn } from "@/utils";
 
+const MAX_PAGE_COUNT = 3;
+const DEFAULT_SIZE = 6;
+
 type PaginationControlsProps = {
   currentPage: number;
+  totalPage: number;
+  currentPath: string;
+  size?: number;
 };
 
-function Pagination({ currentPage }: PaginationControlsProps) {
+function Pagination({
+  currentPage,
+  currentPath,
+  totalPage,
+  size,
```

Comment:
P5

```suggestion
  size = 6,
```

상수를 굳이 만들지 않아도 될 것 같습니다!

Reply:
그래도 나중에 다른사람이 보기에 .. 살짝 고민할 수도 있겠다 싶어서..... ㅠㅠ